systest/multi-tenancy/upgrade_test.go

```diff
@@ -0,0 +1,65 @@
+//go:build upgrade
+
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/dgraph-io/dgraph/dgraphtest"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+const (
+	srcDB, dstDB = "v22.0.2", "v23.0.0-rc1"
+)
+
+type MultitenancyTestSuite struct {
+	suite.Suite
+	dc dgraphtest.Cluster
+	lc *dgraphtest.LocalCluster
+}
+
+func (suite *MultitenancyTestSuite) SetupTest() {
+	conf := dgraphtest.NewClusterConfig().WithNumAlphas(1).WithNumZeros(1).WithReplicas(1).
+		WithACL(20 * time.Second).WithEncryption().WithVersion(srcDB)
+	c, err := dgraphtest.NewLocalCluster(conf)
+	x.Panic(err)
+	x.Panic(c.Start())
+
+	suite.dc = c
+	suite.lc = c
```

Comment:
If we are setting them to the same value, what's the difference exactly?

Reply:
We need access to both the interfae object and concrete object because we want to setup the cluster for both integration and upgrade test. The concrete object is useful in files with upgrade tag so that we can call upgrade function, the interface object is useful for doing things in team tests because they are common for both integration and upgrade.